app/helpers/locations_helper.rb

```diff
@@ -32,8 +32,17 @@ def display_attributes(location)
       { name: "Common type of unit", value: location.type_of_unit },
       { name: "Mobility type", value: location.mobility_type },
       { name: "Code", value: location.location_code },
-      { name: "Availability", value: "Available from #{location.available_from.to_formatted_s(:govuk_date)}" },
+      { name: "Availability", value: location_availability(location) },
       { name: "Status", value: location.status },
     ]
   end
+
+  def location_availability(location)
+    availability = "Active from #{location.available_from.to_formatted_s(:govuk_date)}"
+    location.location_deactivations.each do |deactivation|
+      availability << " to #{(deactivation.deactivation_date - 1.day).to_formatted_s(:govuk_date)}\nDeactivated on #{deactivation.deactivation_date.to_formatted_s(:govuk_date)}"
```

Comment:
just to flag you’ve seen that I’ve converted Dates to Times so we actually save UTC midnight rather than local midnight in the database for our date - this might cause an issue here? https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/980/files#diff-bee6109a705a78cdfa12162d723fe642c7b7c8e53045513503b6d8e574e41b15R206

Reply:
Hmm, I think it should still be fine, but I'll rebase and check when you get your PR merged in! Thanks!